presto-docs/src/main/sphinx/release/release-0.215.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.215
+=============
+
+General Changes
+---------------
+
+* Add dynamic schedule support for grouped execution . When a stage can be executed
+  with grouped execution and the stage has no remote source, table partitions can be
+  scheduled to tasks in a dynamic way. This can mitigate skew for grouped executed queries.
+  The dynamic schedule feature for grouped execution must be enabled with the
+  ``dynamic_schedule_for_grouped_execution`` config property or the
+  ``dynamic-schedule-for-grouped-execution`` config property.
+* Enable spatial joins for join conditions expressed with :func:`ST_Within`
+* Improve spatial join performance by accelerating build-side geometries.
+* Add input validation checks to :func:`ST_LineString` to conform with the specification
+* Add :func:`beta_cdf` and :func:`inverse_beta_cdf` functions.
+* Split reporting of raw input data and processed input data for source operators.
+* Correctly report processed input data for source stages in EXPLAIN ANALYZE.
+* Don't report raw input data for Values, Local Exchange and Local Merge So
+* Simplify EXPLAIN (TYPE IO) output when there are too many discrete components.
+* Adding multiple null convention support for polymorphic scalar functions.
+* Bugfix: Fail query that uses non-leaf group so that it does not hang forever.
```

Comment:
We don't start with "Bugfix" for fixes.
We can say "Fail queries that use non-leaf resource groups to prevent them from getting stuck." or sth like that.